browser-test/src/support/admin_questions.ts

```diff
@@ -1216,9 +1225,10 @@ export class AdminQuestions {
 
     await this.expectAdminQuestionsPageWithCreateSuccessToast()
 
-    expectedQuestionText = expectedQuestionText ?? questionText
-
-    await this.expectDraftQuestionExist(questionName, expectedQuestionText)
+    if (!markdown) {
```

Comment:
question (blocking): Why did you wrap this in `!markdown`? The way this is intended to work is you pass in `expectedQuestionText` if it's a markdown question and it checks for that first. https://github.com/civiform/civiform/pull/6688/files#diff-23ad99c6f330f49fe04c7f59414c9dd6933551c37ea33586d1094e92e1dcf016R1222

Reply:
Because it's difficult to assert the markdown text without some additional manipulation which I am afraid would add brittleness to the test. For example, for the currency question the expected text is "*currency question*", with the asterisks because that's the pre-markdown rendering, and the received text is "currency question" because the markdown has now become stylistic rendering.

We have tests asserting the appearance of the markdown text in screenshots and I don't think we're losing much by skipping the innerText assertions (especially considering it would be onerous to assert that markdown is rendered correctly via presence of HTML elements).

Reply:
That makes sense! In that case, can you remove the `expectedQuestionText` param and logic (it's only used [here](https://github.com/civiform/civiform/blob/15c8a10338b236b07db5767b27f88bbdea318b25/browser-test/src/applicant/navigation/application_navigation_with_eligibility_conditions.test.ts#L359) I think) since we don't need it anymore?

Reply:
Alternatively you could pass in "currency question" for `expectedTest` if we wanted to keep this check, but I don't feel tied to that since we have other tests and it seems like it's not super clear how to use this workaround I added.

Reply:
done! removed the expectedQuestionText usage since that felt cleaner